File: qlty-cli/tests/lang/java/basic.in/Parameters.java
class Parameters {
    public static void f0() {
    }

    public static void f1(Object dog, Object cat) {
    }

    public static void f2(Object a, Object b, Object c, Object d, Object e, Object f) {
    }

    public static void f3() {
        Object foo = bar(1, 2, 3, 4);
    }

    public static Object bar(int a, int b, int c, int d) {
        return new Object();
    }
}

File: qlty-cli/tests/lang/csharp/basic.in/Parameters.cs
using System;

public class Parameters
{
    public static void F0()
    {
    }

    public static void F1(object dog, object cat)
    {
    }

    public static void F2(object a, object b, object c, object d, object e, object f)
    {
    }

    public static void F3()
    {
        object foo = Bar(1, 2, 3, 4);
    }

    public static object Bar(int a, int b, int c, int d)
    {
        return new object();
    }
}
